images/cassandra/configs/latest.apko.yaml

```diff
@@ -0,0 +1,50 @@
+contents:
+  packages:
+    - cassandra
+    - python3
+    - bash 
+    - busybox
```

Comment:
Just double checking we need these?  I don't see an entrypoint script so maybe we dont?

Reply:
resolving as duplicate of https://github.com/chainguard-images/images/pull/1164#discussion_r1281070260

Reply:
Well as much I remember, busy box was needed as explained [here](https://github.com/chainguard-images/images/pull/1164#discussion_r1281074699), I will check bash once again and if not required, will remove it